ndrive.spec

```
@@ -45,6 +49,17 @@ excludes = [
 ]
 
 data = [(data, "data")]
+hiddenimports = []
+
+hiddenimports += [
+    f"nxdrive.dao.migrations.engine.{migration.with_suffix('').name}"
+    for migration in engine_migrations.glob("[!_]*.py")
+]
+hiddenimports += [
+    f"nxdrive.dao.migrations.manager.{migration.with_suffix('').name}"
+    for migration in manager_migrations.glob("[!_]*.py")
+]
```

That could be simplified as:
```suggestion
hiddenimports = [
    migration.as_posix().replace("/", ".")
    for migration in Path("nxdrive/dao/migrations").glob("**/[0-9]*.py")
]
```